src/table/README.md

```diff
@@ -0,0 +1,175 @@
+# Table Component
+
+Component to render tabular data.
+
+## Features
+
+* Sorting
+
+## Currently Unsupported
+
+* Pagination
+* Filtering
+* Fixed row
+* Fixed column
+* Striped rows
+* Hoverable row styling
+* Multi-column sort
+* Row headers
+* Expandable row
+
+## Usage
+
+### Basic usage
+
+```javascript
+import React from 'react';
+import {Table} from 'baseui/table';
+
+const data = [
+  ['1', 'Sarah', 'Brown', 31, 'New York No. 1 Anywhere'],
+  ['2', 'Jane', 'Smith', 32, 'San Francisco No. 1 Anywhere'],
+  ['3', 'Joe', 'Black', 33, 'Sydney No. 1 Anywhere'],
+  ['4', 'Jane', 'Red', 34, 'London No. 1 Anywhere'],
+ ].map(row => [`${row[1]} ${row[2]}`, row[3], row[4]]); // selects data to display
+
+export default = () => <Table columns={['Name', 'Age', 'Address']} data={data}/>,
+```
+
+### Advanced Usage
+
+#### Custom rendering
+
+```javascript
+import {
+  StyledRoot
+  StyledHead
+  StyledHeadCell
+  StyledBody
+  StyledRow
+  StyledCell
+} from 'baseui/table';
+
+const data = [
+  [1, 'John', 'Smith'],
+  [2, 'Lois', 'Lane'],
+];
+
+export default () => (
+  <StyledRoot>
```

Comment:
I know we have `Roots` in most of components but it seems odd to not call it `StyledTable` here? It could make the code more readable since it would be more obvious that it's a table.

Reply:
fixed in 49c9094349f1c078d78fd8f397d09871166c9cda